### PR TITLE
(v0.9.8-release) Update testenv.properties

### PR DESCRIPTION
**Pull request by llxia**

### CODE REVIEW

**Comment by llxia** [comment]
https://ci.adoptium.net/view/Test_grinder/job/Grinder/7327/

**Comment by llxia** [comment]
FYI @pshipton @AdamBrousseau

**Review by pshipton — APPROVED** [review]

**Review by karianna — APPROVED** [review]

File: testenv/testenv.properties [hunk 1]
```diff
@@ -14,14 +14,14 @@ JDK11_REPO=https://github.com/adoptium/jdk11u.git
 JDK11_BRANCH=jdk-11.0.20+6
 JDK17_REPO=https://github.com/adoptium/jdk17u.git
 JDK17_BRANCH=jdk-17.0.8+5
+JDK20_REPO=https://github.com/adoptium/jdk20u.git
+JDK20_BRANCH=jdk-20.0.1+7
```
Comment (karianna) [inline testenv/testenv.properties:18]: Is that not a 20.0.2 now?

**Review by karianna — CHANGES_REQUESTED** [review]

File: testenv/testenv.properties (hunk 1 above)
Comment (AdamBrousseau) [reply testenv/testenv.properties:18]: Not until July 18 ;)

**Review by karianna — APPROVED** [review]

**Pull request merged**